lib/Workflow/GlobalAction.php

```diff
@@ -108,11 +108,11 @@ public function getGuard(): ?string
 
     /**
      * @param Workflow $workflow
-     * @param mixed $subject
+     * @param object $subject
      *
-     * @return bool
+     * @return mixed
      */
-    public function isGuardValid(Workflow $workflow, $subject): bool
+    public function isGuardValid(Workflow $workflow, $subject): mixed
```

Comment:
Should an "isser" really return `mixed`? It seems strange to me, I'd always assume it returns a `bool`.

Reply:
@jdreesen I changed it as "mixed", because evaluateExpression() returns "mixed"

https://github.com/pimcore/pimcore/pull/12401/files/4ce0015c82735579a17ea5648b973e3bd4cc847a#:~:text=%2D%3EgetGuard())%3B-,return%20%24this%2D%3EexpressionService%2D%3EevaluateExpression(%24workflow%2C%20%24subject%2C%20%24this%2D%3EgetGuard())%3B,-%7D

Reply:
@jdreesen I will check it.